### PR TITLE
first draft commit for collaboration page

### PR DESCRIPTION
**Pull request by lhawthorn**
add all the different collaboration spaces along with TODOs for updates (calendar link, email list names, etc.)

### CODE REVIEW

File: Collaboration.md [hunk 1]
```diff
@@ -1,7 +1,87 @@
...
-creating now to park in YouTube links section
+## Getting Started with InstructLab
+
+If you need help getting started with using or contributing to InstructLab, the best way to do so is via our [project Slack workspace](#chat) or [email lists](#list-catalog) rather than posts on social media.   
+
+## [Project Meetings](#Project-Meetings)
+
+We host weekly community meetings each Tuesday at 14:00 UTC. ([time zone converter](https://www.timeanddate.com/worldclock/meetingdetails.html?year=2024&month=5&day=14&hour=14&min=0&sec=0&p1=37&p2=43&p3=101&p4=224&p5=213&p6=771&p7=248&p8=2)) 
+
+Weekly community meetings will feature regular updates on project happenings, including announcements, demos, places to meet with the project maintainers, and more.
```
Comment (jjasghar) [inline Collaboration.md:13]:
```suggestion
Weekly community meetings will feature regular updates on project happenings, including announcements, demos, places to meet with the project maintainers, and more. 
The second half of the hour will be dedicated to community organization and maintenance. 
We plan on having dedicated Office Hours on Thursday, which will be <TODO>.
```

File: Collaboration.md [hunk 2]
```diff
@@ -1,7 +1,87 @@
...
+
+The project will host more meetings as it evolves. To stay up to date on when meetings are added and how to join them, subscribe to the InstructLab project calendar. 
+
+<b>TODO: Add calendar link and fix spacing after fixing this todo.</b>
+
+## [Chat](#Chat)
+
+For real-time chat discussions, please join our [InstructLab Slack workspace](https://github.com/instructlab/community/blob/main/InstructLabSlackGuide.md).
+
+Slack history is deleted after 90 days, so for conversations that should preserved for a longer period use the project mailing lists.
```
Comment (jjasghar) [inline Collaboration.md:25]:
```suggestion
Slack history is deleted after 90 days, so for conversations that should preserved for a longer period use the project mailing lists. It is good to say that if you want to add feedback or think there is a "large issue" to discuss, the mailing list or the specific repository issue tracker is a good place to put this type of text. If you are unsure of where to put it <users@instructlab.ai> is the best place to start.
```

**Comment by jjasghar** [comment]
A couple suggestions, we can put them where ever :)

**Comment by lhawthorn** [comment]
@jjasghar Can you please review and, if it looks good to you, merge? Thank you!

**Review by jjasghar — APPROVED** [review]

**Pull request merged**